### PR TITLE
[TGBOT-239] Extract service layer to module as persistent service

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-239

### CODE REVIEW

**Review by schneidervs — APPROVED** [review]

**Review by yurasava — APPROVED** [review]

File: persistence-core/src/main/java/com/tutrit/Main.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.tutrit;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
```
Comment (AlexanderYarokhovich) [inline persistence-core/src/main/java/com/tutrit/Main.java:5]: I would rather use a logger

**Review by AlexanderYarokhovich — APPROVED** [review]
lgfm

**Comment by Artem-Metlushko** [comment]
I agree with solve

**Review by Artem-Metlushko — APPROVED** [review]
I agree with code

**Pull request merged**